themes/klasik/js/costume.js

```diff
@@ -0,0 +1,14 @@
+jQuery(document).ready(function() {
```

Comment:
`costume` dalam bahasa Inggris artinya `kostum` atau seragam/pakaian khusus. 
`to customise` dalam bahasa Indonesia adalah `menyesuaikan`. Kata `kustom` tidak ada di KBBI.

Sebaiknya gunakan istilah lain yg lebih tepat untuk nama file ini -- menggunakan istilah Indonesia.